src/main/java/org/jitsi/videobridge/cc/AdaptiveTrackProjection.java

```diff
@@ -113,26 +113,29 @@
      */
     private int targetIndex = RTPEncodingDesc.SUSPENDED_INDEX;
 
-    //TODO(brian): we need this to know which frameprojectioncontext to make
-    // based on the payload type of a packet.  is there a better way?
-    private final Map<Byte, PayloadType> payloadTypes = new HashMap<>();
+    private final Map<Byte, PayloadType> payloadTypes;
```

Comment:
Do we want to declare this as type `ConcurrentMap<Byte, PayloadType>` (and similarly in the constructor and in BitrateController) to ensure that the API enforces the thread-safety we need?

Reply:
I thought it was best to leave that responsibility to the caller and leave this as the more basic type.  At the end of the day this class just expects to be able to read from this map to find things, and there are a couple ways a caller might implement that safely so the flexibility seemed desirable.